sample-plugin/pom.xml

```diff
@@ -1698,7 +1698,7 @@
       <id>2.492.x</id>
       <properties>
         <bom>2.492.x</bom>
-        <jenkins.version>2.492.2</jenkins.version>
+        <jenkins.version>2.492.3</jenkins.version>
```

Comment:
Is 2.492 considered the active or previous lines? I have a feeling this is supposed to be `2.492.1` still...

Reply:
I think we are supposed to update this after the next `.1` release

Reply:
But given we just have a security release yeah we should update this now.